### DISCUSSION TITLE
[Suggestion] Rubber Tapping

### DISCUSSION THREAD

**Discussion by Entralia** [vectorwing/FarmersDelight#560] [Ideas]
Greetings, Farmer's Delight Team!

Thanks kindly for making this wonderful mod -- thoroughly enjoying it!

Just a little suggestion, perhaps explore 'Rubber Tapping' ?
Seems very apt for your quality > quantity approach.

Regardless, thanks kindly for your time & efforts!
Take care, stay safe, and be well :)

Warm Regards,
Entralia

**Comment by vectorwing** [comment]
Hmmm... FD doesn't really have uses for rubber, so I'm not sure about it. Rubber tends to be more technology-oriented (belts, insulation etc). Thanks for suggesting, though!
